content/docs/latest/end2endtests_27845856.md

```diff
@@ -1,72 +0,0 @@
----
-title: "Apache Hive : End2EndTests"
-date: 2024-12-12
----
-
-# Apache Hive : End2EndTests
-
-As discussed in HIVE-2985 (<http://issues.apache.org/jira/browse/HIVE-2985>), the goal of this exercise is to create a end to end test framework.
```

Comment:
I'm considering voiding this ticket because we have not seen any progress for 12 years.
https://issues.apache.org/jira/browse/HIVE-2985